.gitignore

```diff
@@ -69,3 +69,6 @@ Temporary Items
 
 # sftp configuration file
 sftp-config.json
+tests/testrunner/utils/__init__.pyc
+tests/testrunner/utils/RokuUtils.pyc
+tests/testrunner/utils/TestrunnerUtils.pyc
```

Comment:
.pyc files in general shouldn't be committed, this was added as part of #11 being merged, so I think this is now causing a conflict